src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -772,7 +781,13 @@ public function getPhinxType($sqlType)
             case 'text':
             case 'json':
                 return static::PHINX_TYPE_TEXT;
+            case 'smallint':
+                return array(
+                    'name' => 'smallint',
+                    'limit' => static::INT_SMALL
+                );
             case 'int':
+            case 'int2':
```

Comment:
I asked the same question but in the rush to use smallint in my project, I did not analyze precisely because the `getPhinxType()` method, has types that are not used in the PostgreSQL adapter.

Following my analysis:
In commit 7bad4f9d5436be08ae40f98fc77e1b638d780534 @czogori added some types in `getPhinxType()` method that are not returned by `getColumns()` method. I believe that the intention of @czogori was to use the column [udt_name] (http://www.postgresql.org/docs/8.4/static/infoschema-columns.html) of the PostgreSQL table `information_schema.columns`.

At this time, I removed the `int2` and I will make another pull request with a refactor to improve the use of types to postgreSQL, okay? These other types are not implemented in Phinx but may be helpful for future implementation of other types used by PostgreSQL.

Note: `int2` is `udt_name` to smallint in PostgreSQL